### PR TITLE
Use SecureString for passwords in DB commands

### PR DESCRIPTION
**Pull request by openbots-su**

### CODE REVIEW

**Review by openbots-ch — DISMISSED** [review]

File: OpenBots.Commands/OpenBots.Commands.Database/DefineDatabaseConnectionCommand.cs [hunk 1]
```diff
@@ -149,32 +148,6 @@ public override List<Control> Render(IfrmCommandEditor editor, ICommandControls
 			RenderedControls.Add(_connectionString);
 
 			_connectionStringPassword = commandControls.CreateDefaultInputFor("v_ConnectionStringPassword", this);
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Database/DefineDatabaseConnectionCommand.cs:150]: this still needs to be one Inputgroup

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: OpenBots.Commands/OpenBots.Commands.Database/DefineDatabaseConnectionCommand.cs [hunk 2]
```diff
@@ -183,9 +156,7 @@ public override List<Control> Render(IfrmCommandEditor editor, ICommandControls
 			label.Text = "NOTE: If storing the password in the textbox below, please ensure the connection string " +
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Database/DefineDatabaseConnectionCommand.cs:156]: you can move this warning to the Remarks

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: OpenBots.Commands/OpenBots.Commands.Database/DefineDatabaseConnectionCommand.cs (hunk 1 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.Database/DefineDatabaseConnectionCommand.cs]: Addressed in 
d25261a

File: OpenBots.Commands/OpenBots.Commands.Database/DefineDatabaseConnectionCommand.cs (hunk 2 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.Database/DefineDatabaseConnectionCommand.cs]: Addressed in 
d25261a

**Review by openbots-ff — DISMISSED** [review]

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ch — APPROVED** [review]

**Pull request merged**